extensions/gluonTS/build.gradle

```diff
@@ -0,0 +1,36 @@
+group 'ai.dj.gluonTS'
```

Comment:
Although our implementation is based on GluonTS, but our API should be generic for all time series application.

Let's just call our API as: ai.djl.timeseries

and the same for the extension folder name

Reply:
Thanks, I think it's greate. I changed them all to timeseries